packages/config/federated-modules.js

```diff
@@ -0,0 +1,77 @@
+const { resolve } = require('path');
+const { ModuleFederationPlugin } = require('webpack').container;
+
+module.exports = ({
+    root,
+    exposes,
+    shared = [],
+    debug,
+    moduleName
+}) => {
+    const { dependencies, insights } = require(resolve(root, './package.json')) || {};
+    const appName = moduleName || (insights && insights.appname);
```

Comment:
Manged Kafka uses `crc.name`. Which is correct?

Reply:
It should be `insights.appname` this is used widely accross many configs. It's unfortunate that the name changed midway, but switching to a new one would be a few inconveniences.

Reply:
Can't you just short-circuit an `|| crc.name`?

Reply:
We could do that, but the problem is that the `insights.appname` is used widely across plathera configs not only at this place. QE is relying on it as well, couple of other configs (including shared config) uses this constant. We could change it, but that can lead to unwanted bugs without proper investigation where it's used and changing it on all other places. Further this PR does not aim to address this change at all.